docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
     container_name: send-letter-service
     environment:
       - REDIS_ENABLED
-      - REDIS_HOST
+      - REDIS_HOST=send-letter-redis
       - REDIS_PORT
       - REDIS_PASSWORD
```

Comment:
should we provide port and password too? or more explicitly: should we provide port and get rid of password?

Reply:
default values from `application.yaml` are used:
```
  port: ${REDIS_PORT:6379}
  password: ${REDIS_PASSWORD:}
```